src/components/grid/grid-item/grid-item.style.ts

```diff
@@ -67,21 +39,65 @@ export function getSpacing(prop) {
     }
   }
 
-  return prop;
+  return String(prop);
 }
 
-const paddingPropTypes = filterStyledSystemPaddingProps(
-  styledSystemPropTypes.space
-);
+interface GridProperties extends PaddingProps {
```

Comment:
You could export this and then import and extend it in the `GridItemProps` interface (component file) so we're not duplicating etc? Again not a big deal so happy to leave it as is if you feel strongly